src/meta/meta_bulk_load_service.cpp

```diff
@@ -1651,9 +1657,28 @@ void bulk_load_service::try_to_continue_app_bulk_load(
                                different_status_pidx_set));
 }
 
+// ThreadPool: THREAD_POOL_META_SERVER
+/*static*/ bool
+bulk_load_service::validate_ingest_behind(const std::map<std::string, std::string> &envs, bool val)
+{
+    bool app_allow_ingest_behind = false;
+    const auto &iter = envs.find(replica_envs::ROCKSDB_ALLOW_INGEST_BEHIND);
+    if (iter != envs.end()) {
+        if (!buf2bool(iter->second, app_allow_ingest_behind)) {
```

Comment:
Could you reuse func `check_bool_value` here?

Reply:
It's good idea, but I tried and found out that couldn't reuse `check_bool_value` function.
https://github.com/XiaoMi/rdsn/blob/08af400e3b5a0b43c61baa1cb9e4c4dc8f3bbb64/src/meta/app_env_validator.cpp#L118-L126
`check_bool_value` function only check if the result is bool or not, can not get the result value.